README.md

```diff
@@ -94,6 +94,8 @@ There is an active community of Wagtail users and developers responding to quest
 
 For topics and discussions that do not fit Stack Overflow's question and answer format, we have a [Slack workspace](https://github.com/wagtail/wagtail/wiki/Slack) and a [Wagtail Support mailing list](https://groups.google.com/forum/#!forum/wagtail). Please respect the time and effort of volunteers by not asking the same question in multiple places.
 
+[![Join slack community](.github/join-slack-community.png)](https://wagtail.org/slack)
```

Comment:
Could we link this to https://github.com/wagtail/wagtail/wiki/Slack instead? I'd rather not give people a route to Slack that bypasses the instructions about the code of conduct and using #support properly.